### PR TITLE
CIV-11603 Fast Track Claimant Intent

### PR DESCRIPTION
**Pull request by kenneth-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-11603

### Change description ###
Changes for submission for fast track claimant intent

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by portizdev — DISMISSED** [review]

**Review by jeswanth-hmcts — APPROVED** [review]

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField-R2-CUI-nonprod.json [hunk 1]
```diff
@@ -2443,5 +2443,29 @@
...
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "applicant1FullDefenceConfirmAmountPaidSpec",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "APP-SOL-UNSPEC-PROFILE",
+          "APP-SOL-SPEC-PROFILE",
```
Comment (neeta-hmcts) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseField-R2-CUI-nonprod.json:2454]: do we need these user roles ?

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField-R2-CUI-nonprod.json [hunk 2]
```diff
@@ -2443,5 +2443,29 @@
...
+          "CITIZEN-CLAIMANT-PROFILE"
+        ],
+        "CRUD": "CRU"
+      },
+      {
+        "UserRoles": [
+          "caseworker-civil-admin",
+          "RES-SOL-ONE-UNSPEC-PROFILE",
+          "RES-SOL-ONE-SPEC-PROFILE",
+          "RES-SOL-TWO-SPEC-PROFILE",
```
Comment (neeta-hmcts) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseField-R2-CUI-nonprod.json:2464]: same as above

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField-R2-CUI-nonprod.json (hunk 1 above)
Comment (kenneth-hmcts) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseField-R2-CUI-nonprod.json]: Updated

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField-R2-CUI-nonprod.json (hunk 2 above)
Comment (kenneth-hmcts) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseField-R2-CUI-nonprod.json]: Updated

**Review by neeta-hmcts — APPROVED** [review]

**Review by paul-pearson — APPROVED** [review]

**Pull request merged**